__tests__/stream.test.ts

```diff
@@ -226,4 +249,32 @@ describe('Websocket stream history sync', () => {
       mockServer.stop();
     }
   }, 10000);
+
+  it('should ignore unknown stream ids', async () => {
```

Comment:
question: I'm under the impression that even if the wallet fails to handle the unknown stream id we would not be able to validate this in this test. Is there a more explicit way to check the results of these unknown stream ids?

Reply:
As discussed, this test provides enough confidence on the results of the websocket with the balance validation.
Future possibilities can be discussed later.